charts/internal/shoot-system-components/charts/csi-alicloud/templates/auth/csi-snapshot-controller-rbac.yaml

```
@@ -20,13 +20,13 @@ rules:
   verbs: ["get", "list", "watch"]
 - apiGroups: ["snapshot.storage.k8s.io"]
   resources: ["volumesnapshotcontents"]
-  verbs: ["create", "get", "list", "watch", "update", "delete"]
+  verbs: ["create", "get", "list", "watch", "update", "delete", "patch"]
```

In comparison to provider-aws the permissions for `volumesnapshotcontents/status` are missing. Can you check that?
```suggestion
  verbs: ["create", "get", "list", "watch", "update", "delete", "patch"]
- apiGroups: ["snapshot.storage.k8s.io"]
  resources: ["volumesnapshotcontents/status"]
  verbs: ["patch"]
```